tests/model/test_model.py

```diff
@@ -628,6 +628,33 @@ def test_modernize_message_response(self, model, messages,
         assert (model.modernize_message_response(messages)
                 == expected_messages_response)
 
+    @pytest.mark.parametrize('feature_level, to_vary_in_initial_data', [
+        (None, {}),
+        (27, {}),
+        (53, {
+            'max_stream_name_length': 60,
+            'max_topic_length': 60,
+            'max_message_length': 10000,
```

Comment:
This looks good, but we want to ensure that these are not the same as the defaults, to ensure that we process the values correctly - so maybe values relative to the old defaults?